go.mod

```diff
@@ -3,10 +3,16 @@ module github.com/spiffe/spiffe-helper
 go 1.13
 
 require (
-	github.com/andres-erbsen/clock v0.0.0-20160526145045-9e14626cd129
+	github.com/google/go-cmp v0.3.0 // indirect
 	github.com/hashicorp/hcl v1.0.0
-	github.com/spiffe/go-spiffe v0.0.0-20190922191205-018e7197ed1c
-	github.com/spiffe/spire v0.0.0-20191007205405-52bcf313f4f6
-	github.com/stretchr/testify v1.4.0
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/spiffe/go-spiffe v1.1.0
```

Comment:
Hmm, where are we still relying on v1? If possible we should get rid of this dependency.

Reply:
We are using `github.com/spiffe/go-spiffe/spiffetest` which is pulling in this dependancy.

Reply:
dependency removed